### PR TITLE
net8 + handlers + cleanup

### LINKED ISSUE #15: .net8 borders ItemTemplate doesn't show if it includes a border
The ItemTemplate shows blank in coverflow view when a border is used in the data template in .NET 8. 
Using a normal CarouselView works completely fine.

I have attatched the sample repo with frame replaced by border in PanCardViewSample.Views.CoverFlowSampleXamlView.
It also has a CarouselView abouve the CoverFlowView to compare.

https://github.com/Zack-G-I-T/MAUI.NET8.BorderIssue

### PR DESCRIPTION
**Pull request by PavloLukianets**
Probably fixes #15, #16, #17, #18 

@AndreiMisiukevich here you are mate - whats left is to upload to nuget)
[CardsView.Maui.0.0.5.nupkg.zip](https://github.com/AndreiMisiukevich/CardView.MAUI/files/13557548/CardsView.Maui.0.0.5.nupkg.zip)

### CODE REVIEW

File: PanCardView/Common/CardsView.cs [hunk 1]
```diff
@@ -1720,7 +1711,7 @@ private void SendChildToBackIfNeeded(View view, View topView)
 
             if (currentIndex < backIndex)
             {
-                ExecutePreventException(() => LowerChild(view));
+                ExecutePreventException(() => view.ZIndex--);
```
Comment (AndreiMisiukevich) [inline PanCardView/Common/CardsView.cs:1714]: That's doesn't work really.
Do you think we can assume it's a MAUI bug?

Comment (PavloLukianets) [reply PanCardView/Common/CardsView.cs:1714]: dunno) Didn't think up of any substitution, maybe you have any ideas?

Comment (AndreiMisiukevich) [reply PanCardView/Common/CardsView.cs:1714]: Nooo... dude, TBH I am exhausted with all these MAUI issues.
XamarinForms were way more "stable" if we can say so :)

Comment (AndreiMisiukevich) [reply PanCardView/Common/CardsView.cs:1714]: I am gonna merge this PR anyway as it fixes other issues (I hope so)

Comment (PavloLukianets) [reply PanCardView/Common/CardsView.cs:1714]: > Nooo... dude, TBH I am exhausted with all these MAUI issues. XamarinForms were way more "stable" if we can say so :)

true. Well, let's wish the MAUI team, nothing more we can do))

**Pull request merged**